plugin/src/main/scala/migrate/ScalaMigratePlugin.scala

```diff
@@ -0,0 +1,101 @@
+package migrate
+
+import sbt.Keys._
+import sbt._
+import sbt.plugins.JvmPlugin
+import java.nio.file.{ Files, Path, Paths }
+
+import migrate.interfaces.Migrate
+
+import scala.collection.JavaConverters._
+import buildinfo.BuildInfo
+
+case class Scala3Inputs(scalacOptions: Seq[String], classpath: Seq[Path], classDirectory: Path)
+
+object ScalaMigratePlugin extends AutoPlugin {
+  val classpathAttribute      = AttributeKey[Seq[Path]]("unmanagedClasspath")
+  val scalacOptionAttribute   = AttributeKey[Seq[String]]("scalacOptions")
+  val classDirectoryAttribute = AttributeKey[Path]("scala3ClassDirectory")
```

Comment:
We need only one attribute to store the inputs.
```scala
scala3inputsAttirbute = AttributeKey[Scala3Inputs]("scala3Inputs")
```

Reply:
yes ^^ Good catch